src/renderer/webapi/apis/FilesApi.ts

```diff
@@ -0,0 +1,72 @@
+import { callBlobApi, callJsonApi } from "./callHttpApi";
+import { WebAPIClient } from "../WebAPIClient";
+import * as path from "path";
```

Comment:
Is this a node module? Then we must not use it in the UI! If this is ok, move it to top.